hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -2927,11 +2927,11 @@ public static boolean isAclEnabled(Configuration conf) {
   // Web Proxy Configs
   ////////////////////////////////
   public static final String PROXY_PREFIX = "yarn.web-proxy.";
-  
+
```

Comment:
These changes are unrelated to the content we modified. We shouldn't alter this section.